doc/source/user_guide/howto/create_input_file.rst

```diff
@@ -1,20 +1,38 @@
 .. _input_file_for_pyacp:
 
 Create input file for PyACP
----------------------------
+===========================
 
 To start working with PyACP, an input file that contains the mesh is required. PyACP supports reading
-the mesh from CDB and DAT files with the :meth:`.ACPWorkflow.from_cdb_or_dat_file()` method. PyACP reads the mesh, including any coordinate systems, element sets,
-edge sets, and materials from the input file. Once the layup has been created with PyACP, you can export its CDB file with the :meth:`.ACPWorkflow.get_local_cdb_file` method. This file
-contains all the data from the initial input file along with the layup information and
-materials added by PyACP. An attempt is made to preserve the original input file as much as possible.
-This includes the original mesh, materials, and boundary conditions. Therefore, you may directly use the exported CDB file
-for an analysis through PyMAPDL. For more information, see :ref:`pymapdl_workflow_example`.
+the mesh from:
 
-.. _input_file_from_mechanical:
+#. CDB or DAT files, using either the :meth:`.ACPWorkflow.from_cdb_or_dat_file()` method, or the :meth:`.ACPInstance.import_model()` method with the ``"ansys::cdb"`` or ``"ansys:dat"`` format.
+#. HDF5 transfer files generated by Mechanical, using the :meth:`.ACPWorkflow.from_mechanical_h5_file()` method or the :meth:`.ACPInstance.import_model()` method with the ``"ansys::h5"`` format.
 
-Create an input file with Ansys Mechanical
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+PyACP reads the mesh, including any coordinate systems, element sets, edge sets,
+and materials from the input file. Once the layup has been created with PyACP, you
```

Comment:
Note that the materials are not part of the HDF5 transfer from Mechanical or refer to the section below.